dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Machines/ReactivePowerControlLoops/ReactivePowerControlLoop.mo

```diff
@@ -63,6 +63,12 @@ model ReactivePowerControlLoop "Simplified Reactive Power Control Loop model"
   parameter Types.ReactivePowerPu QStator0Pu "Start value of the generator stator reactive power in pu (base QNomAlt) (generator convention)";
   parameter Types.VoltageModulePu UStatorRef0Pu "Start value of the generator stator voltage reference in pu (base UNom)";
 
+// Limiter for UStatorRefPu
```

Comment:
These two parameters, should be put with the other parameters. You can find the coding style in Dynawo/UsersGuide/NamingConventions

Reply:
Also, you can add a default value of 1.15 for max and 0.85 for min, thanks